sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ShouldImplementExportedInterfaces.cs

```diff
@@ -91,4 +91,18 @@ class ExportingMembers_Are_Ignored
         [Export(typeof(Exported))]
         public NotExported MyMethod() { }
     }
+
+    interface ISomething<T> { }
+    public class BaseThing { }
+
+    [Export(typeof(ISomething<BaseThing>))]
+    public class BaseSomethingImplementation : ISomething<BaseThing>
+    {
+
+    }
+
+    [Export(typeof(ISomething<>))]

```

Comment:
For the avoidance of doubt, this is an invalid export isn't it?

Reply:
@duncanp-sonar Not sure whether things changed but this wasn't a "native" supported syntax but this was supported through some extension (see https://blogs.msdn.microsoft.com/gblock/2009/03/21/why-doesnt-mef-support-open-generics-for-exports-because-mef-is-not-type-based/)

Reply:
Reading again the article I am actually not sure this works.